.github/workflows/ingestion-deploy-dev.yaml

```diff
@@ -14,7 +14,9 @@ jobs:
     uses: mbta/workflows/.github/workflows/deploy-ecs.yml@main
     with:
       app-name: lamp-ingestion
-      environment: 'dev'
+      environment: dev
+      dockerfile-path: ./py_gtfs_rt_ingestion/
```

Comment:
define the dockerfile path. the default was `.`, so this needed to change when we moved it into the subdirs.